Governance.md

```diff
@@ -18,103 +18,163 @@ Specific goals include:
+- Identify bugs, issues and feature requests from users and turn them into issues on GitHub
+- Review bug reports and features and make sure that they are valid and contain sufficient detail to implement
+- Prioritize bug reports and features so that contributors and other developers know which issues are most important and in need of contribution
+- Read the communication channels relevant to their area(s)
+- Assist with testing new releases of the software
+- Identify contributors who would be helpful as supporters; nominate these contributors to the PMC
 
-- Ensure that votes are taken on issues that require votes
-- Ensure that the activities of the BOINC community are in agreement with this document and established procedures
-- Schedule monthly meetings of the PMC
+Supporters can resign by sending an email to the PMC chair or the PMC public email list.
 
-The Chair remains in that role until they retire or the PMC votes to remove them. He or she has no additional authority beyond that of other PMC members.
+### 2.5 Project Management Committee
+The Project Management Committee (PMC) is a group of community members who engage with and play a leadership role on the BOINC project.  They are selected on the basis of one or more of the following criteria:
+- Directly contributing in any of the ways listed in [section 2.2 Contributors](#22-contributors)
+- Operating a related system, such as an account manager, that is used by a significant number of volunteers
+- Operating a project with a significant number of volunteers
+- Contributing significant resources to the BOINC project, for example by paying the salary of a contributor
+- Thought leaders in the field of high performance computing who have an interest in promoting the use of volunteer computing
 
-### 2.5 Official Record
-The official list of committers and members of the PMC shall be maintained at http://boinc.berkeley.edu/trac/wiki/ProjectGovernance. This page shall also clearly list the email address of the PMC public email list with instructions on how someone can subscribe to the list.
+The functions of the PMC are:
+- Decide on the strategic directions of the project
+- Decide issues of intellectual property (copyright, licensing) and other legal issues
+- Support and encourage the recruitment and development of committers and supporters
+- Ensure that necessary tasks for the long term success of the BOINC project are being done
+- Resolve conflicts within the community
+- Review and vote on nominated committers and supporters
+- Vote on adding or removing members of the PMC as needed
+- Decide on the set of “approved” projects and account managers             
+- Modify the governance policies of the BOINC project as needed
+- And any other such other thing as might be required from time to time     
+
+Individual PMC members are expected to actively participate in these processes by:
```

Comment:
Perhaps useful to specify that PMC members are expected to participate in [some percentage of] monthly PMC calls?

Reply:
I now see in 2.5.3 "Inactive Removal", language saying "any member of the PMC that goes for a year without participating in at least one PMC meeting should be considered for removal from the PMC".  So maybe here it's only necessary to say "Regularly participate in in monthly PMC calls" (*this being the goal; 2.5.3 being the condition for automatic termination)

Reply:
I've added the statement "Regularly participate in monthly PMC calls"

Reply:
2.5 and 2.5.3 are now enough out of synch to introduce confusion.  I'd suggest you revise 2.5.3 to use the "Regularly participate" expression as well.  

Ie "As a result, any member of the PMC that does not regularly participate in PMC meetings should be considered for removal from the PMC"

(*If you want to lay out an explicit baseline,  once per year seems a bit too low a bar?)

Reply:
Btw, I'm not clear if it's preferably for me to comment here in the discussions, or to fork and propose changes in a pull request? -- I've assumed here as the idea is to have a single pull request on which we can vote.  But let me know, for future reference.

Reply:
I've updated the text (see the new section here: https://github.com/BOINC/boinc-policy/blob/knr_supporters/Governance.md#253-inactive-removal).  Y

Reply:
You can create a pull request against this branch which would be a great way to propose changes.  It would probably be easier than commenting inline.

Reply:
Thx.